src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -448,6 +450,12 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         constructor.newInstance()
     }
 
+    public Object createExecCallback(OutputStream out, OutputStream in) {
+        Class callbackClass = loadClass('com.github.dockerjava.core.command.ExecStartResultCallback')
+        Constructor constructor = callbackClass.getConstructor(OutputStream.class, OutputStream.class)
```

Comment:
No need to put '.class' at the end. Groovy will automagically use the "class" if you pass "OutputStream, OutputStream".